logic/levels/dungeon01.py

```diff
@@ -8,14 +8,11 @@
         'H': 'HergonScout',
         'M': 'Monokh'
     },
-
-	
-	
     'chars': [
         {
             'class': 'G',
             'type': 'enemy',
-            'position': (13, 30),
```

Comment:
The character will get stuck in the textures with y coord less than 12. Please fix.